src/zookeeper_config.py

```diff
@@ -91,4 +107,19 @@ def create_properties(self, config):
             if config.get(k, None) is not None:
                 props += f"\n{v}={config[k]}"
 
+        if self.ssl_enabled():
+            props += TLS_ZOOKEEPER_PROPERTIES
+            props += "\nzookeeper.connect="
+            for host in cluster.active_hosts:
+                props += f"{host}:2182,"
+
         return props
+
+    def ssl_enabled(self):
```

Comment:
Although this is obviously fine, could we add a test for this to avoid regression?